src/Session/SessionDonation/SessionObjects/Objects.php

```diff
@@ -0,0 +1,12 @@
+<?php
+namespace Give\Session\SessionDonation\SessionObjects;
+
+interface Objects {
```

Comment:
@ravinderk Let's move this to the `Give\Session` root and call it `SessionObject`. I suspect it will be useful in other domains.

Reply:
fixed: https://github.com/impress-org/givewp/compare/89e8238e2fa8...c14c8ac2fdf6